simple/rdm_rma.c

```diff
@@ -182,11 +159,12 @@ static int run_test(void)
 	}
 	clock_gettime(CLOCK_MONOTONIC, &end);
 
-	if (machr)
-		show_perf_mr(transfer_size, iterations, &start, &end, 1, g_argc,
-			       	g_argv);
+	if (opts.machr)
+		show_perf_mr(opts.transfer_size, opts.iterations, &start, &end, 
+				2, opts.argc, opts.argv);
 	else
-		show_perf(test_name, transfer_size, iterations, &start, &end, 1);
+		show_perf(test_name, opts.transfer_size, opts.iterations, 
+				&start, &end, 2);
```

Comment:
you're right. I'll change it to 1 and check other examples if there's same situation.